docs/asciidoc/migration/v5-to-v6.adoc

```diff
@@ -46,3 +46,10 @@ Just simply use the dozer-core:
     <version>6.0.0</version>
 </dependency>
 ----
+
+=== 4. XSD namespace and location has been modified
```

Comment:
Can you remove this change. This doesnt apply to v6, it applies to v6.1, as per:
- https://github.com/DozerMapper/dozer/blob/master/docs/asciidoc/migration/v6-to-v61.adoc